doc/optimizers.md

```diff
@@ -1394,6 +1394,114 @@ double Optimize(arma::mat& X);
  * [Semidefinite programming on Wikipedia](https://en.wikipedia.org/wiki/Semidefinite_programming)
  * [Semidefinite programs](#semidefinite-programs) (includes example usage of `PrimalDualSolver`)
 
+## Quasi-Hyperbolic Momentum Update
+
+ *An optimizer for [differentiable separable functions](#differentiable-separable-functions).*
+
+ Quasi Hyperbolic Momentum Update is an update policy for SGD where the Quasi Hyperbolic terms are added to the
+ parametrisation. Simply put QHM’s update rule is a weighted average of momentum’s and plain SGD’s
+ update rule.
```

Comment:
This is a bit confusing for users since it's not described as an optimizer.  I would suggest something like this:

```
Quasi-hyperbolic momentum update SGD (QHSGD) is an SGD-like optimizer with momentum where quasi-hyperbolic terms are added to the parametrization.  The update rule for this optimizer is a weighted average of momentum SGD and vanilla SGD.
```